docs/_summary.md

```diff
@@ -40,6 +40,7 @@
   * [Unit Testing](unit_testing.md)
   * [Useful Functions](ref_functions.md)
   * [Configurator Support](reference_configurator_support.md)
+  * [info.json Format](reference_info_json.md)
```

Comment:
This same change needs to be made to `docs/_sidebar.md`. This file only updates the GitBook docs.

Reply:
_sidebar.md isn't there anymore. I assume someone finally cleaned that up.

Reply:
Ahh, you're right. Jack deleted it in #4661.